### DISCUSSION TITLE
Image truncated

### DISCUSSION THREAD

**Discussion by Showmetheflower** [CancerDataScience/CrowdsourcingDataset-Amgadetal2019#10] [Q&A]
hello,when I run the download_crowdsource_dataset.py,it raise image file is truncated (27 bytes not processed).And I add "from PIL import ImageFile  ImageFile.LOAD_TRUNCATED_IMAGES = True'' these code,the download picture is damaged.Can you help me ?

**Comment by kheffah** [comment]
I'm not 100% sure what's the source of the error. I have a feeling this is caused by a network timeout when the image was being downloaded. Could you provide more context? Does this happen with the RGB or the mask, and which one?

**Reply by Showmetheflower** [reply]
RGB.And the mask is all black.

**Reply by kheffah** [reply]
OK, if this happens with the RGB then it's quite likely that this is a network timeout. In other words, the download did not complete and if you re-start the download for that RGB it should be OK assuming you have good internet. 
As for the mask, it looks black because the pixel values encode class. So a value of 1 means tumor, for example. So the contrast is very low. If you use matplotlib to view the mask, for example, you'd see that pixels have different values.

**Reply by Showmetheflower** [reply]
My network has always been poor, can you provide a link similar to a network disk or email so that I can download pictures

**Reply by kheffah** [reply]
@Showmetheflower If you have access to the TCGA slides, which can be publicly obtained from many sources, you can use openslide to obtain the RGBs corresponding to the masks of interest offline. The masks themselves should be downloadable using this repo as-is and don't require a fast connection.
